public/app/plugins/datasource/loki/querySplitting.ts

```diff
@@ -131,10 +131,10 @@ export function runGroupedQueries(datasource: LokiDatasource, requests: LokiGrou
       .runQuery({ ...requests[requestGroup].request, range, requestId, targets })
       .subscribe({
         next: (partialResponse) => {
-          if (partialResponse.error) {
-            subscriber.error(partialResponse.error);
-          }
           mergedResponse = combineResponses(mergedResponse, partialResponse);
+          if (responseHasErrors(mergedResponse)) {
+            shouldStop = true;
+          }
```

Comment:
How do we let the user know there was an error?

Reply:
> To further ellaborate on my first comment, I ask: why 1 and not 2. I understand the case for syntax errors, but sometimes queries fail due to timeouts or server-side conditions, in those cases doing 1 will imply losing the results we successfully got so far, and focusing on the error response.
> 
> It would be ideal to be able to stop on syntax errors, and attempt to continue with other mundane errors, but I'm not sure it's possible.

i'm fine with doing [2], i'll update the PR 👍

Reply:
it happens automatically by the code that calls "us".

as long as the `DataQueryResponse`'s `.error` or `.errors` attribute is not empty, it will show a red error message. (and it will also show any dataframes that are there)